### ISSUE TITLE
Affiliation with the Haskell Foundation

### ISSUE THREAD

**Issue by emilypi** [haskell/cabal#7385]
Hi All,

I'd like to formally invite the Cabal team and its advisors to collaborate directly with the Haskell Foundation as affiliates. As many of you know, the HF has been quite successful in bringing many distant parts of the community together to get them talking, collaborating, and generally working together. I think this a great opportunity for Cabal. Particularly, because Cabal is an integral part of the tooling system, which affects everyone else. 

## What does Affiliation Entail? 

It's a pledge to abide by the Guidelines For Respectful Communication (GRC) defined [here](https://haskell.foundation/guidelines-for-respectful-communication/#:~:text=We%20strive%20to%20treat%20every%20person%20with%20respect.&text=Even%20if%20we%20occasionally%20doubt,or%20harassment%20in%20our%20communication.), and an invitation to join us in our Slack to help plan our technical agendas, and to communicate with all of our other affiliates. 

A slack link may be found here: https://join.slack.com/t/haskell-foundation/shared_invite/zt-mjh76fw0-CEjg2NbyVE8rVQDvR~0F4A

Additionally, the HF makes sure that any resource requests (e.g. maintainers, hardware etc.) are handled appropriately. If Cabal is having trouble locating people to maintain, we can help! The HF does not expect to dictate or change the way maintenance is done in any way, nor dictate the course of projects - only make sure that they are well-supported and happy, and if there is a particular feature request in our agenda, we work with existing tooling to avoid writing a brand-new solution. We focus on honoring community contributions instead of building our own from scratch. 

The full details may be found here: https://haskell.foundation/en/affiliates/

## How does Cabal affiliate? 

Just say yes! As of now, Cabal satisfies all the requirements for affiliation. We will provide a link on our website, and a platform for sharing requests and interaction with the rest of the community, and that's it. Easy!

**Comment by emilypi** [comment]
Pinging maintainers+advisors: @dcoutts @hvr @fgaz @gbaz @Mikolaj @davean @ekmett

**Comment by hasufell** [comment]
> The HF does not expect to dictate or change the way maintenance is done in any way, nor dictate the course of projects

From the HF page:

> We invite existing Haskell projects, committees and communities to explicitly affiliate and align themselves with the Haskell Foundation. Affiliation means that the group supports the goals of the Haskell Foundation, and that the Haskell Foundation in turn supports this group.

In the past, the goals of HF weren't particularly clear to me. There's a strong drive to increase adoption in industry (and particularly windows it seems), which is an honorable goal, but what technical trade-offs are expected, what is expected of tools etc seems to be 1) in flux and 2) definitely something HF is going to comment on and write proposals about (e.g. unified installer proposal). I'd expect the HF at some point to write a proposal about installers as well, e.g. formulating their expectations about user experience or technical aspects.

I'm not saying whether I agree with that approach or not, but I think maintainers should be aware of it and the goals should be clarified.

**Comment by gbaz** [comment]
Perhaps the HF page should be fixed to say "broad goals" or "vision" to indicate that it means endorsing e.g. the vision statement: "The Haskell Foundation is an independent, non-profit organization dedicated to broadening the adoption of Haskell, by supporting its ecosystem of tools, libraries, education, and research."

I.e. even if the HF were to want to write proposals about specific things related to the same domain as an affiliated project, I _hope_ the onus would not fall on such affiliated projects to a priori now have to support such proposals as "goals of the Haskell Foundation." If my understanding is wrong, that should be clarified asap, and if it is correct, then we should clean up the language of HF affiliation to prevent misunderstandings.

**Comment by philderbeast** [comment]
https://haskell.foundation/affiliates/ includes cabal. Can we close this issue now?

![image](https://github.com/haskell/cabal/assets/633283/e643f6e7-2be5-40a9-9e26-ad796bf36881)

**Comment by ulysses4ever** [comment]
I think to fully address this issue, Cabal's own website/README should mention the affiliation.

**Comment by philderbeast** [comment]
> I think to fully address this issue, Cabal's own website/README should mention the affiliation.

Cabal's own website is [haskell.org/cabal](https://www.haskell.org/cabal/) isn't it, as linked to from the *About* on [github.com/haskell/cabal](https://github.com/haskell/cabal)?

![image](https://github.com/haskell/cabal/assets/633283/c9c6df56-97d1-4952-99a4-1e4f9bf35edf)

**Comment by ulysses4ever** [comment]
Correct. Does it mention the affiliation?

**Comment by philderbeast** [comment]
> Correct. Does it mention the affiliation?

No it doesn't mention the affiliation. Some of this page seems out of date such as the [Update Troubleshooting](https://www.haskell.org/cabal/#troubleshooting) section saying that v2/new commands are new-ish.

**Comment by ulysses4ever** [comment]
Yeah, the website needs more love in general. It's also in GitHub I think.
